outputs.tf

```diff
@@ -66,5 +66,10 @@ output "forseti-server-storage-bucket" {
 
 output "suffix" {
   description = "The random suffix appended to Forseti resources"
-  value       = "${local.random_hash}"
+  value = "${local.random_hash}"
+}
+
+output "forseti-version" {
```

Comment:
Since we're hardcoding the forseti version in the tests we no longer need this output.